docs/recipes/plugin/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "plugin-lighthouse-recipe",
```

Comment:
hmm. what does this package.json do? do we need it?

Reply:
> what does this package.json do?

it lets you run the recipe

Reply:
ah. it's a little confusing to me with two manifests over here. Like.. for a user.. do they need both? 

the plugin folder doesnt have a readme, but anyone making a plugin would be placing a readme in there.

so i'm thinking we just move this readme a level deeper and drop this package.json

Reply:
> so i'm thinking we just move this readme a level deeper and drop this package.json

this makes sense, so here it is, but it basically isn't runnable now